features/data/dependency_monkey/flask/pipeline.yaml

```diff
@@ -5,24 +5,13 @@ boots:
   name: PythonVersionBoot
 - configuration: {}
   name: RHELVersionBoot
-- configuration:
-    default_platform: linux-x86_64
-  name: PlatformBoot
 - configuration: {}
   name: FullySpecifiedEnvironment
 sieves:
 - configuration: {}
   name: CutPreReleasesSieve
-- configuration: {}
-  name: Functools32BackportSieve
-- configuration: {}
-  name: Enum34BackportSieve
-- configuration: {}
-  name: MockBackportSieve
```

Comment:
Just for the reference - these pipeline units were replaced with prescriptions, so we do not have any Python implementation for them.